projects/core/src/cart/facade/cart.service.ts

```diff
@@ -126,29 +120,33 @@ export class CartService {
   }
 
   addEntry(productCode: string, quantity: number): void {
-    this.store
-      .pipe(
-        select(fromSelector.getActiveCartState),
-        tap(cartState => {
-          if (!this.isCreated(cartState.value.content) && !cartState.loading) {
-            this.store.dispatch(
-              new fromAction.CreateCart({ userId: this.cartData.userId })
-            );
-          }
-        }),
-        filter(cartState => this.isCreated(cartState.value.content)),
-        take(1)
-      )
-      .subscribe(_ => {
-        this.store.dispatch(
-          new fromAction.AddEntry({
-            userId: this.cartData.userId,
-            cartId: this.cartData.cartId,
-            productCode: productCode,
-            quantity: quantity,
-          })
-        );
+    if (!CartUtilService.isCreated(this.cartData.cart)) {
```

Comment:
It's possible that customer uses `for` loop to `addEntry`. If so, there will be many carts created.